scripts/linters/pre_commit_linter.py

```diff
@@ -324,10 +326,21 @@ def _get_file_extensions(file_extension_type):
         all_file_extensions_type: list(str). The list of all file extensions
         to be linted and checked.
     """
-    all_file_extensions_type = ['.js', '.py', '.html', '.css']
+    all_file_extensions_type = ['*.js', '*.py', '*.html', '*.css']
+
+    # Check if '*.js' and '*.ts' both are present in file_extension_type.
+    js_and_ts_is_present = '*.js' in file_extension_type and (
+        '*.ts' in file_extension_type)
+
+    if js_and_ts_is_present:
+        python_utils.PRINT(
+            'Please either use "*.js" or "*.ts" both are not '
```

Comment:
or "*.ts". Both are not allowed together.


(Btw ... why do we have this restriction?)

Reply:
Done.